src/LinkDotNet.Blog.Web/ConfigurationExtension.cs

```diff
@@ -0,0 +1,105 @@
+using System;
+using Amazon.Util.Internal.PlatformServices;
+using LinkDotNet.Blog.Domain;
+using LinkDotNet.Blog.Web.Authentication.OpenIdConnect;
+using LinkDotNet.Blog.Web.Features.ShowBlogPost.Components;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+
+namespace LinkDotNet.Blog.Web;
+
+public static class ConfigurationExtension
+{
+    public static IServiceCollection AddConfiguration(this IServiceCollection services)
```

Comment:
I am leaning towards only exposing `AddConfiguration` and `AddConfiguration` calls then all the other methods. What is your take?

Reply:
ya I was thinking the same, moved everything inside.